### DISCUSSION TITLE
recent change of behavior of insert on empty node

### DISCUSSION THREAD

**Discussion by code77** [freeplane/freeplane#1361] [Help] [answered]
Hello,

Over the years of using Freeplane I have a habit of creating a new node by pressing INS and pasting the content of the clipboard into a newly created node. Because the node was empty, Freeplane interpreted it as **editing**. 
**It looks that this behavior changed**. Now any paste operation on a node, even empty, results in a **creation of a new child nod**e.

I've tried looking for a configuration to bring back the previous behavior but I couldn't find any. There is _Settings->Behaviour->On key type->Override content_ that works when somebody start typing when a node is selected. I guess I'm looking for an equivalent when Paste operation is done on an empty node.

**Comment by dpolivaev** [comment, marked as answer]
The behavior for paste is reverted in the recently released version 1.11.6. I think you have not updated yet, have you?
The default shortcut for new nodes has changed because nowadays not every keyboard has an insert key.
Now users who prefer using the insert key can assign it to the new child menu item themselves.

**Comment by code77** [comment]
I was on 1.11.5
All works. Dimitry, you're the best!
